mindsdb_native/libs/controllers/transaction.py

```diff
@@ -282,6 +317,48 @@ def run(self):
 
                     output_data[f'{predicted_col}_confidence'][row_number] = probability_true_prediction
 
+            # confidence estimation
```

Comment:
It might make more sense for this bit here to be under it's own phase or under the `ModelAnalysis` phase, but there's a bunch of older logic in here for which this is true, so let's ignore this for now.